extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -476,8 +490,11 @@ export class AmpAnalytics extends AMP.BaseElement {
       this.remoteConfig_['transport']['iframe'] = undefined;
     }
 
+    typeConfig = expandConfigRequest(typeConfig || {});
```

Comment:
Why isn't it `const typeConfig = expandConfigRequest(this.predefinedConfig_[type] || {})` above?

Reply:
mostly because I need to check `if (typeConfig)` , but anyway fixed